src/app/query/bitcoin/ordinals/brc20/brc20-tokens.hooks.ts

```diff
@@ -92,18 +94,24 @@ function makeBrc20Token(token: Brc20Token) {
     marketData: token.min_listed_unit_price
       ? createMarketData(
           createMarketPair(token.ticker, 'USD'),
-          createMoney(new BigNumber(token.min_listed_unit_price), 'USD')
+          createMoney(
+            new BigNumber(token.min_listed_unit_price).times(
+              baseCurrencyAmountInQuote(createMoney(1, 'BTC'), btcMarketData).amount
+            ),
+            'USD'
+          )
```

Comment:
@kyranjamie is this the best way to handle a price in sats using the `createMarketData`?